.travis.yml

```diff
@@ -6,19 +6,16 @@ addons:
   apt:
     packages:
       - google-chrome-stable
+  sauce_connect: true
 before_script:
   - 'export DISPLAY=:99.0'
   - sh -e /etc/init.d/xvfb start &
+  - npm run http-server &
   - sleep 3
+script: npm run ci
 env:
   global:
     - secure: >-
-        mBBiPIc2161EK7HWl8LkoaJF073cyvBaTLKx55nYiN/EKi1adx+JLN9j5frH4CvqB+0nmSum3ltC7ONQA+nqr4jq0qXc5s1cdAHDkKupxdUFrLBALaYVIHXRrJVr+Ew6y3EWX+8dUf9843j0ToGIFhAaRIYQ1X+X37rPJGbNi7UzvJPn8634tBvIt+cjQzaS8mFfXy3BbAfFS4Qm92DXekTBinJIuZbBJ8bxzgMi0PZ6bgWHJ8v5d/vdEEyYkM3zs995bgOPf/UWuUfR5akdIu07pgN7Z6ZgzIKus0r6zECmV6fvgyJWkoMXboM1MoWToti8n8P6k8V2QhQHkOYTP7Zea3PL9MJEO3txd7/9cujZ0EGIkYFZBgfRsxyJJgi+gphWgxEIy4AGoyHKjPlsj3pyXs1QoScjIVw4ecz9bNI9YLcKZbaRCSe5CcZeR0i/V0dFb35Cn+/AS59CbVGyLpXj2GDDseNe6L0cU2y2Xy6mIVR7djH/twLlsInc1Q+IICTBPOrO1vN8bu4wtDyT/R81tOQJdvYUwkBmqzTQjhNx4hmDiLB6vGQhpzkpcGKEyddFSxR3oxvdk2wNo73JZfyVjx+CMLr/s90wI67bGXJTykeOgoc/zbUS8JW6X/Uf7mgdKF9dlwMiegcbh7qcGcs1BoQnOXVAk7T1K6hwOFg=
+        a7ksz7FbsFIH1iLlx7SFGNv4A3ovFrDNowtfFIFl1Ns193vjlo+nNw7LoR3pscUpSqUzz9bgwsL36TeS4wX6mtKivHybke7j02XuIfr14tK2GckD3C2zVbxK2Qvhcfx39nHxweJ7t43xyDS+nbFL+bJnlpaNT8yYIlWWdgaSdEGttuiXsGYN9Lf/qgiwBR9fyfi90hNFmWHzp1t36H9+f9tjIEvYuv779d1Xaodg4kxv984ZIQJHzQ0n0KFKnsG4djnQ2aRBATuIq9A874VxFG9fr6MbnSgIu9mdVQmaqbxWICeI3iJwhjYvx/iloQyJ+vu2ZM6FeAexZXdr1WbnfyrsroKkE7jlbJIuzhDQPj+Mo3cW/DfOBEf4xN4v/4o7XNE8/Y1nubEcKkOnJmVRsprLY5NYUsjsu5yPx/UQ1eOkqo3fYIj/hotCar/53ZFhXlNunGnod3QWMagydpFmrY8cprPPrL8YinjWsH8wo9Q+hFFJLkcaIdF4I4fyGIdbRlrNoLEV7/YJG25KwNIurRwKe+D/EoHWUeJpwf8kI89nZH/RA1ioqZSoZqRWBP4Ayddym4wtaattgfjrO12DcuGUDn0hSBcC+Xyz6DecIeVGMDRn/EUUP7mzuBhXs8Fyh88WdyWMZHWW1fXKe43C7vOUh2Z8ehSjcKLw4rQwoTE=
```

Comment:
So we have to check-in keys to use Saucelabs? Feels a little strange and dangerous but I haven't really delt with Saucelabs yet.